src/simsopt/geo/curvefilament.py

```diff
@@ -0,0 +1,265 @@
+import simsoptpp as sopp
+from .._core.graph_optimizable import Optimizable
+from .._core.derivative import Derivative
+from .curve import Curve
+from .jit import jit
+import numpy as np
+import jax.numpy as jnp
+from jax import vjp, jvp
+
+
```

Comment:
How about adding a module docstring saying that everything in this file is related to finite-build coils. This isn't obvious from the filename `curvefilament.py`. Maybe also consider a filename that makes this more clear, like `multifilament.py` or `finite_build.py`?